index.bs

```diff
@@ -6536,8 +6536,11 @@ engine.
     setting the `extraData` parameter to the digest of |attToBeSigned| using the hash algorithm corresponding to the "alg" signature algorithm.
     (For the "RS256" algorithm, this would be a SHA-256 digest.)
 
-    Set the |pubArea| field to the public area of the credential public key, the |certInfo| field to the output parameter of the
-    same name, and the |sig| field to the signature obtained from the above procedure.
+    Set the |pubArea| field to the public area of the credential public key (the TPMT_PUBLIC structure), the |certInfo| field (the TPMS_ATTEST structure)
+    to the output parameter of the same name, and the |sig| field to the signature obtained from the above procedure.
+
+    Note: If the |pubArea| is read from the TPM using the TPM2_ReadPublic command, that command returns a TPM2B_PUBLIC structure. TPM2B_PUBLIC
```

Comment:
This Note does not seem relevant to RP processing, which what this section is really all about.

Reply:
Not really - this is in the signing procedure, which is performed by the authenticator. The RP part is below in the verification procedure.

Reply:
Fair - withdrawn.